### DISCUSSION TITLE
Blank screen on launching SEB

### DISCUSSION THREAD

**Discussion by nputhuva-rm** [SafeExamBrowser/seb-win-refactoring#819] [General]
I downloaded seb 3.6.0 code from https://github.com/SafeExamBrowser/seb-win-refactoring.git 
tried creating build without any code changes. build creation was success but on opening the browser after installing it, just a blank screen is displaying
on pressing ctrl+q  and enter, i'm able to exit the screen
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/104403591/8037e540-3b1e-48b1-bfb2-247b2285d85f)
[2024-03-01_14h16m55s_Browser.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/14458744/2024-03-01_14h16m55s_Browser.log)
[2024-03-01_14h16m55s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/14458745/2024-03-01_14h16m55s_Client.log)
[2024-03-01_14h16m55s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/14458746/2024-03-01_14h16m55s_Runtime.log)

The application installed using Setup\bin\x64\Debug\Setup.msi works fine
But application installed using Setup\bin\x64\Release\Setup.msi, SetupBundle\bin\x64\Debug \SetupBundle.exe, and SetupBundle\bin\x64\Release \SetupBundle.exe are having the issue

Could someone please suggest a solution? @dbuechel

**Comment by dbuechel** [comment]
Make sure you always rebuild the solution before trying to debug it. Apart from that, we unfortunately cannot give support on general issues with customizations, that is reserved for members of the SEB Alliance: https://safeexambrowser.org/alliance/fees.html.

**Reply by dbuechel** [reply]
See #821 for the actual solution.
